ARCs/arc-0200.md

```diff
@@ -0,0 +1,232 @@
+        { "type": "uint256", "name": "value_" }
+      ],
+      "returns": { "type": "bool", "desc": "Success" }
+    },
+    {
+      "name": "allowance",
+      "desc": "Returns the current allowance of the spender of the tokens of the owner",
+      "readonly": true,
+      "args": [
+        { "type": "address", "name": "owner_" },
+        { "type": "address", "name": "spender_" }
+      ],
+      "returns": { "type": "uint256", "desc": "The remaining allowance" }
+    }
+  ],
+  "events": [
+    {
+      "name": "Transfer",
+      "desc": "Transfer of tokens",
+      "args": [
+        {
+          "type": "address",
+          "name": "from_",
+          "desc": "The source of transfer of tokens"
+        },
+        {
+          "type": "address",
+          "name": "to_",
+          "desc": "The destination of transfer of tokens"
+        },
+        {
+          "type": "uint256",
+          "name": "value_",
+          "desc": "The amount of tokens transferred"
+        }
+      ]
+    },
+    {
+      "name": "Approval",
+      "desc": "Approval of tokens",
+      "args": [
+        {
+          "type": "address",
+          "name": "owner_",
+          "desc": "The owner of the tokens"
+        },
+        {
+          "type": "address",
+          "name": "spender_",
+          "desc": "The approved spender of tokens"
+        },
+        {
+          "type": "uint256",
+          "name": "value_",
+          "desc": "The amount of tokens approve"
+        }
+      ]
+    }
+  ]
+}
+```
+
+Ownership of a token by a zero address indicates that a token is out of circulation indefinitely, or otherwise burned or destroyed.
+
+The methods `transfer` and `transferFrom` method MUST error when the balance of `from_` is insufficient.
+The `transferFrom` method MUST error unless called by an approved spender as defined by an extension defined in this ARC.
+The methods `transfer` and `transferFrom` MUST emit a `Transfer` event that a transfer is successful.
+A `Transfer` event SHOULD be emitted, with `from_` being the zero address, when a token is first minted.
+A `Transfer` event SHOULD be emitted, with `to_` being the zero address, when a token is destroyed.
+
+The `Approval` event MUST be emitted when the `approve` method is called successfully.
+
+A value of zero for the `approve` method and the `Approval` event indicates no approval.
+When a `Transfer` event emits following the `transferFrom` method, this may also indicate that the approved value for the token is decremented.
+The contract MUST allow multiple operators per owner.
+
+All methods in this standard that are marked as `readonly` MUST be read-only as defined by [ARC-22](./arc-0022.md).
+
+## Rationale
+
+This specification is based on <a href="https://eips.ethereum.org/EIPS/eip-20">ERC-20</a>.
+
+### Core Specification
+
+The core specification identical to ERC-20 with the exception of:
+
+- Method and event arguments are appended, rather than prefixed, with \_.
```

Comment:
I don't think there's any reason to do this in an ARC.  There is no semantic meaning to the variable names here - if some HLL compiled the json down to something that could be called from the HLL and use the argument names from the standard, it ought to be the convention of the language in question how to handle name conflicts.

Reply:
I agree. It doesn't matter because when it comes does to it names are not translated in to the ABI where it counts. I'm going to revert this to the original state, ie. _from _to just like the ERC-20 spec.

Reply:
What's the reason for `_` at all? (outside of aligning with ERC20)

Reply:
Yes. I will exterminate the tick ; )